services/booking-api/lambdas/create.js

```diff
@@ -0,0 +1,18 @@
+import { throwError } from '@helsingborg-stad/npm-api-error-handling';
+import to from 'await-to-js';
+
+import * as response from '../../../libs/response';
+
+import booking from '../helpers/booking';
+
+export async function main(event) {
+  const body = { ...JSON.parse(event.body) };
+
+  const [error, createBookingResponse] = await to(booking.create(body));
+  if (error) {
+    throwError(error);
+  }
+
+  const { data } = createBookingResponse.data;
+  return response.success(200, data);
```

Comment:
We're trying to avoid using naming like 'data' because all thing is data and it is in some regards unclear what it is.
Maybe it could be named 'calendarBooking' or something in that direction? :)

Reply:
This is destructured from the Datatorget response. We are not naming it to data ourself.

Reply:
To ease the readability, i would suggest just naming the data object or the destructed data attribute to something mer descriptive. The reason is just to ease the understanding of what the data attribute contains.

My suggestion if it i'm understanding the code correct is that the data attribute is a booking object, then maybe an improvement could be
```JS
const { data: booking } = createBookingResponse.data
```